packages/composites/anchor/src/endpoint/price/bluna.ts

```diff
@@ -0,0 +1,45 @@
+import { AdapterContext, AdapterRequest, AdapterResponse, InputParameters } from '@chainlink/types'
+import { Config } from '../../config'
+import * as view from '@chainlink/terra-view-function-adapter'
+import { Validator } from '@chainlink/ea-bootstrap'
+
+export const FROM = 'BLUNA'
+export const INTERMEDIARY_TOKEN_DECIMALS = 8
+export const INTERMEDIARY_TOKEN = 'LUNA'
+
+export const inputParameters: InputParameters = {
+  terraBLunaContractAddress: true,
+}
+
+export const execute = async (
+  input: AdapterRequest,
+  context: AdapterContext,
+  _: Config,
+  taAdapterResponse: AdapterResponse,
+): Promise<AdapterResponse> => {
+  const validator = new Validator(input, inputParameters)
+  if (validator.error) throw validator.error
+  const _config = view.makeConfig()
+  const _execute = view.makeExecute(_config)
+  const viewFunctionAdapterRequest: AdapterRequest = {
+    id: input.id,
+    data: {
+      address: validator.validated.data.terraBLunaContractAddress,
+      query: {
+        state: {},
+      },
+    },
+  }
+  const viewFunctionAdapterResponse = await _execute(viewFunctionAdapterRequest, context)
```

Comment:
Could we type this response?

Reply:
isn't it already implicitly typed from the function call's response?

<img width="742" alt="Screen Shot 2021-12-06 at 1 32 01 PM" src="https://user-images.githubusercontent.com/24400947/144925958-b3228f64-706a-4711-a8e6-a53acdaa5a3a.png">

Reply:
AdapterResponse has an any type on `data`. 

I'll handle this in my types OKR though to make it a generic type.